manufacturing/internals/filter_panel/README.md

```diff
@@ -60,13 +44,37 @@ BEFORE purchasing any parts.**
 [a8mcmc]: https://www.mcmaster.com/92095A193/
 [a9mcmc]: https://www.mcmaster.com/92095A189/
 
-## Assembly
+## Custom parts
+
+This is the design for the custom cut acrylic plate for the filter housings.
+
+
+
+| Acrylic plate |
+:------------------|
+|![.iam](images/filter_panel_acrylic.jpg)|
+
+| Download Type | Link   |
+|:--------------|:-------|
+| Source: Inventor part file | [.ipt](filter_panel_acrylic_plate.ipt) |
+| Drawing                    | [.idw](filter_panel_acrylic_plate.idw) [.pdf](filter_panel_acrylic_plate.pdf)|
+| Export for laser cutting   | [.dxf](filter_panel_acrylic_plate.dxf) |
+
+The plate may be produced using a laser cutter or with common woodworking tools.
+
+
+## Filter Panel Assembly
+
+| |
+:------------------|
+![.iam](images/filter_panel_assembly_dwg.jpg)|
```

Comment:
I would put this next to the `panel1.jpg` picture from below. Would look more balanced.

A second copy of `panel2` is I guess no longer needed.

Reply:
I'm with it, that makes sense